spec/fixtures/files/plum_records/b9e8325e-baf2-45e4-b32c-5e5b3755c8ef.jsonld

```diff
@@ -0,0 +1,155 @@
+   ],
+   "barcode":"32101086454731",
+   "label":"Folder 193",
+   "is_part_of":"Latin American Ephemera",
+   "coverage":[  
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/e36dd185-6875-4efd-a62d-81f01a700097",
+         "@type":"skos:Concept",
+         "pref_label":"Venezuela",
+         "in_scheme":{  
+            "@id":"https://figgy.princeton.edu/catalog/22597862-062b-4c91-a6ca-cd73a30aceb1",
+            "@type":"skos:ConceptScheme",
+            "pref_label":"LAE Areas"
+         },
+         "exact_match":{  
+            "@id":"http://id.loc.gov/vocabulary/countries/ve"
+         }
+      }
+   ],
+   "dcterms_type":[  
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/42754ac7-7723-4219-81eb-11085946fdca",
+         "@type":"skos:Concept",
+         "pref_label":"Pamphlets",
+         "in_scheme":{  
+            "@id":"https://figgy.princeton.edu/catalog/e7771cb0-e5a9-4ef7-b7ea-8e2f9b103302",
+            "@type":"skos:ConceptScheme",
+            "pref_label":"LAE Genres"
+         },
+         "exact_match":{  
+            "@id":"http://id.loc.gov/vocabulary/graphicMaterials/tgm007415"
+         }
+      }
+   ],
+   "origin_place":[  
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/e36dd185-6875-4efd-a62d-81f01a700097",
+         "@type":"skos:Concept",
+         "pref_label":"Venezuela",
+         "in_scheme":{  
+            "@id":"https://figgy.princeton.edu/catalog/22597862-062b-4c91-a6ca-cd73a30aceb1",
+            "@type":"skos:ConceptScheme",
+            "pref_label":"LAE Areas"
+         },
+         "exact_match":{  
+            "@id":"http://id.loc.gov/vocabulary/countries/ve"
+         }
+      }
+   ],
+   "language":[  
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/1bb6e264-8b3a-452c-9bc5-10e1c4c98c3e",
+         "@type":"skos:Concept",
+         "pref_label":"Spanish",
+         "in_scheme":{  
+            "@id":"https://figgy.princeton.edu/catalog/f0608fb8-d182-437f-b489-8296dcb87678",
+            "@type":"skos:ConceptScheme",
+            "pref_label":"LAE Languages"
+         },
+         "exact_match":{  
+            "@id":"http://id.loc.gov/vocabulary/iso639-1/es"
+         }
+      }
+   ],
+   "subject":[  
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/fb359d07-e82d-479b-b3b9-06dcaebebd9f",
+         "@type":"skos:Concept",
+         "pref_label":"National parks and reserves",
+         "in_scheme":{  
+            "@id":"https://figgy.princeton.edu/catalog/64275d5f-7148-4e3a-b836-084dff3977f0",
+            "@type":"skos:ConceptScheme",
+            "pref_label":"Environment and ecology"
+         },
+         "exact_match":{  
+            "@id":"http://id.loc.gov/authorities/subjects/sh85090045"
+         }
+      },
+      {  
+         "@id":"https://figgy.princeton.edu/catalog/a51c7f0b-4203-4ae9-b063-3e03fac1832b",
+         "@type":"skos:Concept",
+         "pref_label":"Advertising--Tourism",
```

Comment:
I think this is a separate issue, but this looks like a new wrinkle on the subject/category display issue.  Should this display as "Tourism--Advertising" or "Advertising--Tourism" or something else?

Reply:
I understood this to be the actual subject, such that the combined category / subject would be "Tourism -- Advertising--Tourism".

Reply:
That seems silly to me.  But it's possible that is how it should be displayed, or that this is just a data problem — there are two other terms in that vocab that have dashes in them (Tourism--Economic aspects
Tourism--Social aspects).

I'll create a separate issue to handle those, and not worry about the display of these terms until we resolve that.

Reply:
This is the label as it exists in the LAE Subjects fixture, so I think that's a call for them.

Reply:
I've created https://github.com/pulibrary/figgy/issues/456 to track this.